modules/ROOT/pages/rtfctl-permissions.adoc

```diff
@@ -0,0 +1,112 @@
+= RTFCTL Kubernetes Permissions
+
+Understanding and enabling the specific permissions required over the Kubernetes resources is crucial for successfully running `rtfctl` commands. Properly configured role permissions are essential for seamless execution of `rtfctl` operations.
+
+The following table lists all the permissions that you configure using K8s RBACs (https://kubernetes.io/docs/reference/access-authn-authz/rbac/[Role based access control^]) in Kubernetes: 
+
+* Namespace +
```

Comment:
Our style is to put the + on a line by itself, which results in more line-spacing.